TheDescent/esd109/The DescentESD109_updated.py

```diff
@@ -0,0 +1,94 @@
+# # Ethan Davis ESD109 The Descent CodeChat Editor
```

Comment:
What is this? I want just one code file with the solution, not two.

Reply:
This looks like you resolved it -- click the "Resolve conversation" below so you can keep track of what's done and what still needs doing.